tests/imposter/private-api.yaml

```diff
@@ -123,6 +123,8 @@ paths:
       responses:
         "200":
           description: "Answered submitted"
+        "404":
+          description: "Does not exist"
         "400":
           description: "Client Error"
           content:
```

Comment:
The status codes that are returned by Groovy, should be added into this file, otherwise the console will print error such as `No response found in specification for POST http://localhost:8080/answer and status code 201`